app/src/main/java/com/microsoft/research/karya/ui/scenarios/imageData/ImageDataFragment.kt

```diff
@@ -141,7 +137,7 @@ class ImageDataFragment : BaseMTRendererFragment(R.layout.microtask_image_data)
             switchToGridView()
         }
 
-        nextBtnCv.setOnClickListener {
+        nextBtnCv.root.setOnClickListener {
```

Comment:
Why are we setting the listener on the root?

Reply:
Previously nextBtnCv used to refer the root-view of the layout  because it is used with <include> in the xml for the same with viewBinding we need to use the root because now nextBtnCv is not a view.

```xml
  <include
       android:id="@+id/nextBtnCv"
       layout="@layout/microtask_common_next_button"
       android:layout_width="0dp"
       android:layout_height="match_parent"
       app:layout_constraintDimensionRatio="1:1"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintHorizontal_bias="0.5"
       app:layout_constraintStart_toEndOf="@+id/playBtn"
       tools:layout_editor_absoluteY="0dp"
 />
          
```